packages/web-app-files/src/components/SideBar/Details/SpaceDetails.vue

```diff
@@ -81,12 +83,16 @@ export default {
         .slice(webDavPathComponents.indexOf(ref.space.id) + 1)
         .join('/')
 
-      const fileContents = yield ref.$client.files.getFileContents(
-        buildWebDavSpacesPath(ref.space.id, path),
-        { responseType: 'arrayBuffer' }
-      )
-
-      spaceImage.value = Buffer.from(fileContents).toString('base64')
+      const fileInfo = yield ref.$client.files.fileInfo(buildWebDavSpacesPath(ref.space.id, path))
+      const resource = buildResource(fileInfo)
+      spaceImage.value = yield loadPreview({
+        resource,
+        isPublic: false,
+        dimensions: ImageDimension.Preview,
+        server: ref.configuration.server,
+        userId: ref.user.id,
+        token: ref.getToken
```

Comment:
since you have `isPublic` hardcoded to `false` you can omit the `token` from the options.

Reply:
Discussed this via chat. I had a public link token in mind. Of course the access token is needed for fetching the preview with the authenticated context. Sorry for the noise.